src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -125,7 +128,7 @@ export class QueryRunner extends Component<
 
   public render() {
     return (
-      <div>
+      <>
```

Comment:
this tag seems to be empty

Reply:
Yeah it is called a [fragment](https://reactjs.org/docs/fragments.html). I didn't want to add extra items to the DOM